src/tess_asteroids/asteroid.py

```diff
@@ -0,0 +1,60 @@
+"""Class to obtain, correct and analyze TESS asteroid data"""
+import lightkurve as lk
+
+import tess_ephem
+import tess_locator
+import tess_cloud
+import tess_backdrop
+
+
+class Asteroid(lk.TargetPixelFile):
```

Comment:
We should consider naming it `MinorPlanet` to avoid confusion if we measure non-asteroid moving objects?

Reply:
Yeah! Names are important, sounds great! We can totally rename the package too